assets/js/logging.js

```diff
@@ -0,0 +1,21 @@
+var colors = require('colors');
```

Comment:
This particular folder is for stuff only used by webpack for compiling the frontend assets (so this is providing logging of that). 

All compiled assets go into a public folder that is included in static dirs. Let me know where is best to put that folder if not project root.